frontend/src/home/SwitchPlanPopper.tsx

```diff
@@ -1,22 +1,40 @@
 import React from "react";
-import { MenuItem, Menu } from "@material-ui/core";
-import KeyboardArrowDownIcon from "@material-ui/icons/KeyboardArrowDown";
-import KeyboardArrowUpIcon from "@material-ui/icons/KeyboardArrowUp";
+import {
+  MenuItem,
+  Menu,
+  Button,
+  IconButton,
+  Snackbar,
+} from "@material-ui/core";
+import {
+  KeyboardArrowDown as KeyboardArrowDownIcon,
+  KeyboardArrowUp as KeyboardArrowUpIcon,
+  Delete as DeleteIcon,
+} from "@material-ui/icons";
 import styled from "styled-components";
 import { connect } from "react-redux";
 import { AppState } from "../state/reducers/state";
 import { Dispatch } from "redux";
 import { getSchedulesFromState, getActiveScheduleFromState } from "../state";
-import { setActiveScheduleAction } from "../state/actions/schedulesActions";
+import {
+  deletePlan,
+  setActiveScheduleAction,
+} from "../state/actions/schedulesActions";
 import { setNamedSchedule } from "../state/actions/scheduleActions";
 import { NamedSchedule } from "../models/types";
 import Loader from "react-loader-spinner";
+import { deletePlanForUser } from "../services/PlanService";
+import { deletePlanId } from "../state/actions/userActions";
+import { Alert } from "@material-ui/lab";
 
-const SwitchPlanContainer = styled.div`
+const SwitchPlanDropdown = styled(Button)`
   display: flex;
   flex-direction: row;
   align-items: center;
   cursor: pointer;
+  width: 158px;
+  height: 34px;
+  padding: 0px 10px;
 `;
 const SpinnerWrapper = styled.div`
```

Comment:
I know it's not your changed code but could you remove the height and width from this wrapper?